dask/array/tests/test_array_core.py

```diff
@@ -928,6 +928,34 @@ def test_store():
     pytest.raises(ValueError, lambda: store([at, bt], [at, bt]))
 
 
+def test_store_regions():
+    d = da.ones((4, 4, 4), chunks=(2, 2, 2))
+    a, b = d + 1, d + 2
+
+    at = np.zeros(shape=(8, 4, 6))
+    bt = np.zeros(shape=(8, 4, 6))
+
+    region = (slice(None,None,2), slice(None), [1, 2, 4, 5])
+
+    #Single region:
```

Comment:
Modified in new comments. Should I have remembered from our previous interaction with h5netcdf. Thanks ;)